### PR TITLE
Migrate Comments, Dummy users, and Votes, to database

### PR DESCRIPTION
**Pull request by carl-mate-sun-asterisk**
**Issue Link**
Asana: 
https://app.asana.com/0/1202718495875213/1202734152805488/f
https://app.asana.com/0/1202718495875213/1202725278795556/f

Definition of Done:

- Admin can temporarily create dummy users
- Admin can create posts with comments (foreign key), users (foreign key), upvote (foreign key)

Steps to test functionality:
1. Go to localhost:8000/admin
2. Credentials: username: team3; password: sunteam3
3. Create a dummy user:
image
4. Create a post:
image
5. Add a comment and link to a user. I linked it to the test user created earlier:
image
6. Add an upvote which is linked to the test user created earlier:
image
5. Post has been created and is successfully linked to the respective tables via foreign key.
image
image

Notes:
1. Not yet integrated to frontend

### CODE REVIEW

File: backend/post/migrations/0007_dummyuser_remove_post_total_votes_vote_comment_and_more.py [hunk 1]
```diff
@@ -0,0 +1,57 @@
+# Generated by Django 4.1 on 2022-08-10 02:33
+
+from django.db import migrations, models
+import django.db.models.deletion
+
+
```
Comment (kentlouise) [inline backend/post/migrations/0007_dummyuser_remove_post_total_votes_vote_comment_and_more.py]: Should only have one new line.

Comment (carl-mate-sun-asterisk) [reply backend/post/migrations/0007_dummyuser_remove_post_total_votes_vote_comment_and_more.py]: After line 57 po? Mag add lang ng new line?

Comment (carl-mate-sun-asterisk) [reply backend/post/migrations/0007_dummyuser_remove_post_total_votes_vote_comment_and_more.py]: Ohh nevermind po. 5 and 6 pala.

Comment (kentlouise) [reply backend/post/migrations/0007_dummyuser_remove_post_total_votes_vote_comment_and_more.py]: @carl-mate-sun-asterisk 
Remove mo lang yong isang new line Sir.

Comment (carl-mate-sun-asterisk) [reply backend/post/migrations/0007_dummyuser_remove_post_total_votes_vote_comment_and_more.py]: The migrations folder is autogenerated po by django upon creation of the models. Is it okay to modify this?

Comment (carl-mate-sun-asterisk) [reply backend/post/migrations/0007_dummyuser_remove_post_total_votes_vote_comment_and_more.py]: @kentlouise Noted po sir. Thank you

Comment (carl-mate-sun-asterisk) [reply backend/post/migrations/0007_dummyuser_remove_post_total_votes_vote_comment_and_more.py]: @kentlouise Removed the line na po. Thank you sir.

**Review by kentlouise — APPROVED** [review]
# Approved #4 
- Code looks good to me.
- Implementation works well.
[5462dcbd-3370-463e-a5c8-5315c0b1f9bf.webm](https://user-images.githubusercontent.com/87509953/183863370-1d45ce65-289f-47f5-aa2b-6524b39bac66.webm)

**Pull request merged**